backend/task.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
You should try using a linter to help clean up your code 
<img width="600" alt="image" src="https://user-images.githubusercontent.com/30481844/152401622-8e1c0835-ed99-4024-83c0-843d1eb1511c.png">